.github/workflows/tests.yml

```diff
@@ -25,7 +25,7 @@ jobs:
         with:
           # All of these options are optional, so you can remove them if you are happy with the defaults
           skip_after_successful_duplicate: 'true'
-          paths: '["src/**/*.hpp", "src/**/*.cpp", "tests/**/*.hpp", "tests/**/*.cpp", "simulations/**/*.hpp", "simulations/**/*.cpp", "**/CMakeLists.txt", "vendor/**", ".github/**"]'
+          paths: '["src/**/*.hpp", "src/**/*.cpp", "tests/**/*.hpp", "tests/**/*.cpp", "simulations/**/*.hpp", "simulations/**/*.cpp", "**/CMakeLists.txt", "vendor/**", ".github/**", "toolchains/**"]'
```

Comment:
Should this be limited to the toolchains relevant for the CI? E.g. so we don't need to rerun tests if we update the Adastra toolchain

Reply:
I agree, will udpate

Reply:
Should we also include Persee toolchains ?

Reply:
Done in 769769005651c0031ce9aff248b068fedac82ced

Reply:
> Should we also include Persee toolchains ?

The GPU one yes I think